### ISSUE TITLE
get_lldp_neighbor

### ISSUE THREAD

**Issue by mailorderbribes** [napalm-automation-community/napalm-huawei-vrp#2]
```
I saw you marked this method as #ok but its not..
the regex is for a CE and not working correctly on VRP

you could use the fixed width method as i used for the  get_lldp_neighbor_detail or with a new regex.

i didnt rewrite it because netbox is not using this mehtod.



---Actual CE output----
<cr1-somecity-randomdc7>display lldp neighbor brief
Local Interface         Exptime(s) Neighbor Interface      Neighbor Device    
-------------------------------------------------------------------------------
10GE1/0/1                     113  Gi7/1                   cr1-somecity-randomdc2.domain.tld
10GE1/0/45                    106  XGigabitEthernet0/0/43  cr2-somecity-randomdc2
10GE1/0/46                    100  10GE1/0/47              cr1-somecity-randomdc3
10GE1/0/47                    110  XGigabitEthernet0/0/4   cr2-somecity-randomdc5
10GE1/0/48                    102  Gi3/15                  cr1-somecity-randomdc2.domain.tld
10GE1/0/48                    109  Gi3/15.2375             cr1-somecity-randomdc2.domain.tld
10GE1/0/48                    119  Gi1/14.2271             cr1-somecity-randomdc5.domain.tld
10GE1/0/48                     91  Gi1/14.2266             cr1-somecity-randomdc5.domain.tld
10GE1/0/48                    105  Gi1/14.2270             cr1-somecity-randomdc5.domain.tld
10GE1/0/48                    112  Gi1/14.2267             cr1-somecity-randomdc5.domain.tld
10GE1/0/48                    108  Gi1/14.2263             cr1-somecity-randomdc5.domain.tld


---Actual vrp output------
<pek-cr2-cis-sik>display lldp neighbor brief
Local Intf    Neighbor Dev             Neighbor Intf             Exptime(s)
XGE0/0/1      PEK-N5000                Eth1/13                   118   
XGE0/0/3      ASW17-sikpek             Eth 0/2                   117   
XGE0/0/4      pek-cr1-cis-sik.as999... Te5/2                     91    
XGE0/0/24     pek-cr1-ren-bat.as999... Gi4/4.1170                95    
XGE0/0/24     pek-cr1-ren-bat.as999... Gi4/4.1176                101   
XGE0/0/34     atopsw2pek               XGE0/0/2                  118   
XGE0/0/37     pek-sw1-cis-sik          XGE0/0/1                  119   
XGE0/0/48     pek-cr1-cis-sik.as999... Te5/4                     105   


Mind, the "Eth 0/2" is screwing up my regex skills so i resorted to fixed witdh" :)

```

**Comment by tkspuk** [comment]
@mailorderbribes 
I was fixed this issue. released in 0.1.4.

pls run `pip install --upgrade napalm-huawei-vrp` to upgrade you local package.

I am looking forward to your feedback.

locus

**Comment by tkspuk** [comment]
I will close this issue since I have not received more questions.
